.env.sampe

```diff
@@ -0,0 +1,4 @@
+POSTGRES_HOST=POSTGRES_HOST
+POSTGRES_DB=POSTGRES_DB
+POSTGRES_USER=POSTGRES_USER
+POSTGRES_PASSWORD=POSTGRES_PASSWORD
```

Comment:
no blank line at the end of file

Reply:
I don't know what to insert in .dockerignore )

Reply:
what about using google ? ))
https://github.com/GoogleCloudPlatform/getting-started-python/blob/main/optional-kubernetes-engine/.dockerignore